.env.example

```diff
@@ -6,42 +6,15 @@ APP_NAME=Meetable
 # Provide the base URL (no trailing slash) to where you've deployed the app
 APP_URL=https://events.example.org
 
-# Provide a URL to a logo to use in the header bar
-# LOGO_URL=/public/example.svg
-# Depending on the image, you may want to set the width and/or height here
-# LOGO_WIDTH=40px
-# LOGO_HEIGHT=40px
-
-FAVICON_URL=/public/favicon.ico
-
-# Whether or not to enable receiving webmention responses
-ENABLE_WEBMENTION_RESPONSES=true
-
-# Show or hide the ticket URL field
-ENABLE_TICKET_URL=true
-
-# Choose whether RSVP names appear in the ics event titles
-# (Most useful for small team installs)
-SHOW_RSVPS_IN_ICS=false
-
-
+# SETUP_DONE=true
```

Comment:
This needs documentation in the `.env.example` file for when someone configures this by hand. In fact it should probably just be left uncommented so that people don't have to do anything with it.

Reply:
It's in the .env.example file, commented out as the setup can uncomment things. I suppose I can add comments, but perhaps I'll let you stabilize what you want to do, then come in once that's done to remove / provide options for all the manual steps.